### DISCUSSION TITLE
Api list rule is not working with this filter however is is working with the view rule

### DISCUSSION THREAD

**Discussion by meekde** [pocketbase/pocketbase#2363] [Q&A]
I have the following table
<img width="1112" alt="Screen Shot 2023-04-26 at 5 06 40 PM" src="https://user-images.githubusercontent.com/93671827/234497116-62689ffc-f5ba-43a8-a8e7-2a89f4a98454.png">

All the crud operations work but running `pb.collection('domains').getFirstListItem<DomainsResponse>(`site="${site}"`, {
    $autoCancel: false,
  });` returns a 404. I do not understand what is wrong with my list rule

<img width="664" alt="Screen Shot 2023-04-26 at 5 07 58 PM" src="https://user-images.githubusercontent.com/93671827/234497234-ae87485d-f520-4e52-af0c-90349f711b62.png">

![Screen Shot 2023-04-26 at 5 17 29 PM](https://user-images.githubusercontent.com/93671827/234499346-aa4abbcc-2832-441b-bac7-3bf25863527e.png)

**Comment by ganigeorgiev** [comment]
Are you sure that you are authenticated and that your user's id matches with the `site.user.id`?

If you are still not able to identify the issue, could you export your collections configuration (Admin UI > Settings > Export  collections)?

**Comment by meekde** [comment]
Yes, the IDs match. Here is a gist of the exported collection (https://gist.github.com/meekscreativelab/1300712658694c5dfd55aa5594666f84)

**Reply by ganigeorgiev** [reply]
I'm not able to reproduce the issue. I've created 2 users with 2 sites (1 record for each user) and multiple domains related to site1 and site2.

I've tested with PocketBase v0.15.1 and JS SDK v0.14.0:

```js
const site = "SITE_ID";

await pb.collection("users").authWithPassword("test@example.com", "1234567890");

const domain = await pb.collection("domains").getFirstListItem(`site="${site}"`);
```

Make sure that you are authenticated and that you are using the correct site id in the filter.

**Comment by meekde** [comment]
I was not calling the `getUserSSR` before that domain query so the user creds were never loaded in from the cookie. Calling that is now working. Thank you for the prompt response.

![Screen Shot 2023-04-26 at 6 21 08 PM](https://user-images.githubusercontent.com/93671827/234514826-d965bdf8-8920-4d4a-af08-2f1ea05dbe2b.png)
